doc/source/advanced-use.rst

```diff
@@ -249,3 +249,30 @@ Now simply create a jail and supply the pkgs.txt file:
 ``iocell create pkglist=/path-to/pkgs.txt tag=myjail``
 
 This will install ``nginx`` and ``tmux`` in the newly created jail.
+
+Using volumes
+-------------
+
+Jails can have shared directories from the host which is implemented using nullfs(5). The idea is based on volumes used in the docker world. Files can also be Volumes and are copied over at jail startup.
+
+The ``volumes`` property has the following syntax:
```

Comment:
My point wasn't that this is not coherent with the documentation, but that it seems the pattern is to use singular forms instead of plural, for example 'ip4_addr' not 'ip4_addrs', 'source_path' not 'source_paths' and so on, and so forth.